src/components/shared/Wrapper.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Container } from "@material-ui/core";
+import { Container, Typography } from "@material-ui/core";
```

Comment:
src/components/shared/Wrapper.js
  Line 2:21:  'Typography' is defined but never used  no-unused-vars